time stamp
int32 goal_stage
float64[4] wheel_speed
float64[4] steering_angle